templates/promotion/promotion.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable function-call-argument-newline */
+/* eslint-disable max-len */
+/* eslint-disable function-paren-newline, object-curly-newline */
+import { div, a } from '../../scripts/dom-helpers.js';
+
+export default async function decorate(doc) {
+  const $page = doc.querySelector('main > .section');
+  const divPromotion = div({ class: 'promotion-section' });
+  const leftSection = doc.querySelector('.left-column');
+  const rightSection = doc.querySelector('.right-column');
+  leftSection.style.display = 'block';
+  rightSection.style.display = 'block';
```

Comment:
Can this not go in the style sheet?

Reply:
somehow, `style = display:none` gets added to left and right column. This has to be overwritten in code for the section to be displayed. 
![Screen Shot 2024-07-25 at 12 16 49 PM](https://github.com/user-attachments/assets/9774a3da-ddcf-4dc5-96eb-cd126848dcb9)